tests/integration/sharding_tests/test_sharding_relations.py

```diff
@@ -50,16 +61,207 @@ async def test_build_and_deploy(ops_test: OpsTest) -> None:
         application_name=SHARD_ONE_APP_NAME,
     )
 
+    # Will be enabled after DPE-5040 is done
+    # await ops_test.model.deploy(
+    #     MONGOS_APP_NAME,
+    #     channel="6/edge",
+    # )
```

Comment:
DPE-5040 is done, but we don't have an image yet because the `test_charm.py` is broken.
Have you pushed the PR that fixes it yet or no?

Reply:
@Gu1nness yes [here](https://github.com/canonical/mongos-k8s-operator/pull/26)

Reply:
@Gu1nness is the mongos charm released?

Reply:
Still not, @Mehdi-Bendriss needs to investigate why the token he provided is expired.